00-base/01-colors/colors-branding.js

```diff
@@ -0,0 +1,79 @@
+const branding = [
```

Comment:
You could write a quick function to generate ranges of these colors... that'd be a lot more clean/easy, especially if you make the util portable, and use it to generate lists like this elsewhere.

Reply:
I _think_ that the designer would rather see these listed out. I could be wrong, but I'll leave that for a future discussion.